lib/spack/spack/test/cmd/diff.py

```diff
@@ -75,5 +75,5 @@ def test_load_first(install_mockery, mock_fetch, mock_archive, mock_packages):
 
     assert len(result['a_not_b']) == 1
     assert len(result['b_not_a']) == 1
-    assert result['a_not_b'][0] == ['variant_value', 'mpileaks debug bool(False)']
-    assert result['b_not_a'][0] == ['variant_value', 'mpileaks debug bool(True)']
+    assert result['a_not_b'][0] == ['variant_value', 'mpileaks debug False']
```

Comment:
I was just thinking we need to update the docs to no longer have the bool piece, and I looked and the code blocks aren't rendering so there must be an issue there too! https://spack.readthedocs.io/en/latest/basic_usage.html?highlight=spack%20diff#spack-diff. I think that would be in scope of this PR.

Reply:
Sorry I should have checked that after my PR - I was expecting the docs builder to give me some error if the formatting was wrong but looks like not! I'll be more observant next time.

Reply:
Good catch.  I updated the docs and fixed the bug, which was separate.  The issue was no space here:

```rst
.. code-block::console
```

vs:

```rst
.. code-block:: console
```